app/controllers/account_controller.rb

```diff
@@ -160,7 +160,7 @@ def verify # :nologin:
           elsif password != confirmation
             @user.errors.add(:password_confirmation,
                              :validate_user_password_no_match.t)
-          elsif password.length < 5 || password.bytesize > 40
+          elsif password.length < 5 || password.size > 40
```

Comment:
At this point is `password` the real password or the hash?  
If it's the hash then can we validate its size here?  Also see same issue in User.